composer.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "pod-point/reviews-php",
+  "description": "A review service for PHP.",
+  "type": "library",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Pod Point Software Team",
+      "email": "software@pod-point.com"
+    }
+  ],
+  "require": {
+    "php": ">=7.1",
+    "guzzlehttp/guzzle": "~5.3|~6.0",
+    "illuminate/support": "5.*",
+    "laravel/framework": "5.*",
```

Comment:
I was checking some other packages that also need the `config_path` helper and they don't include the whole laravel framework. I think we should be more restrictive on the laravel versions that we support laravel 5.5 onwards would be nice because you can use the package discovery https://laravel.com/docs/7.x/packages#package-discovery what do you think?

Reply:
Cool, Just made another push with the framework version set to 5.5 or above. 

I did look at other projects to see if they use `config_path` and include the framework package. And did see they do not but when I tried to use the import like them all I got was errors on the tests saying that the `config_path` function is missing and couldn't get the tests to pass.

Reply:
I don't have any issues 🤔  if I remove Laravel framework from the dependencies, I think you only need :
`
   "illuminate/support": "5.5.*|5.6.*|5.7.*|5.8.*|6.0.*|7.0.*|8.0.*",
    "illuminate/config": "5.5.*|5.6.*|5.7.*|5.8.*|6.0.*|7.0.*|8.0.*",
`